SDWebImage/SDWebImageImageIOCoder.m

```diff
@@ -430,8 +431,8 @@ - (NSData *)encodedDataWithImage:(UIImage *)image format:(SDImageFormat)format {
     }
     
     NSMutableDictionary *properties = [NSMutableDictionary dictionary];
-#if SD_UIKIT
-    NSInteger exifOrientation = [[self class] sd_exifOrientationFromImageOrientation:image.imageOrientation];
+#if SD_UIKIT || SD_WATCH
```

Comment:
This method is avilable for watchOS. So open it to solve potential orientation issue on watchOS.